CanonicalLtd/documentation-builder
# Upgrade to vanilla-docs-theme 1.3.3

Replace the version of docs-vanilla-theme included in inline CSS in the default template with the new vanilla-docs-theme v1.3.3.

QA
--

``` bash
git clone https://github.com/vanilla-framework/vanilla-framework; cd vanilla-framework
git clone -b update-theme https://github.com/nottrobin/documentation-builder
python3 -m venv env
env/bin/pip install ./documentation-builder
env/bin/python ./documentation-builder/bin/documentation-builder --base-directories docs --force
```

Now open `build/en/index.html` in your browser, check it looks okay with the new theme.
